izbor.cpp

```diff
@@ -8,10 +8,18 @@ Izbor::Izbor(QWidget *parent)
     , ui(new Ui::Izbor)
 {
     ui->setupUi(this);
+
+    QObject::connect(this, &Izbor::ucitajKviz, this, &Izbor::ucitavanjeKviza);
+    QObject::connect(this, &Izbor::igraj, this, &Izbor::igranje);
+    QObject::connect(this, &Izbor::zavrsi, this, &Izbor::ispis);
+
     QPixmap slika2("://database/slika2.jpg");
     w = ui->label_slika2->width();
     h = ui->label_slika2->height();
     ui->label_slika2->setPixmap(slika2.scaled(w, h, Qt::IgnoreAspectRatio));
+
+    ui->comboBox_odgovori->setPlaceholderText("--Vas odgovor--");
+    ui->comboBox_odgovori->setCurrentIndex(-1);
```

Comment:
E, sad, pitanje je kako Qt funkcioniše i ti primeri koje si gledala. Neki stil koji sam usvojio je da u konstruktoru uglavnom bude minimalno stvari koje je potrebno inicijalizovati, pritom mislim na objekte.
Sve ove dodatne metode povezivanja signala i slotova, kao i setovanje slike i plejsholdera bi išli u neku posebnu metodu koja bi se pozvala nakon konstruktora.

Možeš ovde pročitati izabrani odgovor (za sad tebi Builder pattern ne treba), ali to je otprilike ideja koja je i meni poznata: https://stackoverflow.com/questions/3905784/what-not-to-do-in-a-constructor